pages/initiatives/gsoc/gsoc2020ideas.md

```diff
@@ -221,3 +221,42 @@ Have a look at our GitHub [wiki page](https://github.com/blabla1337/skf-flask),
 * [Glenn ten Cate](mailto:glenn.ten.cate@owasp.org)
 * [Riccardo ten Cate](mailto:riccardo.ten.cate@owasp.org)
 
+### [OWASP Risk Assessment Framework](https://github.com/OWASP/RiskAssessmentFramework)
+
+##### Explanation of Ideas
+
+The OWASP Risk Assessment Framework consist of Static application security testing and Risk Assessment tools, Eventhough there are many SAST tools available for testers, but the compatibility and the Environement setup process is complex. By using OWASP Risk Assessment Framework's Static Appilication Security Testing tool Testers will be able to analyse and review their code quality and vulnerabilities without any additional setup. OWASP Risk Assessment Framework can be integrated in the DevSecOps toolchain to help developers to write and produce secure code.
+
+##### Expected Results
+* A new features with awesome 
+* Make some improvement in scanner (Sast tool)
+* Add another tool other than sonarqube (Integrate other tools to Raf)
```

Comment:
Should RAF be full caps? (As an acronym)

Reply:
fix it